gapic-generator-cloud/templates/cloud/service/client/_client.erb

```diff
@@ -0,0 +1,110 @@
+<%- assert_locals service -%>
+<% @requires = capture do %>
+require "gapic/common"
+require "gapic/config"
+require "gapic/config/method"
+
+require "<%= service.gem.name.gsub "-", "/" %>"
```

Comment:
There is very little difference in this file from the base file. Do you think it's possible to avoid overriding the whole file for these customizations?

Reply:
Possible, but I don't know what those changes should be just yet. I'd like to get further along getting a real, working cloud gem with all the expected dependencies and then figure out how to avoid duplication.

Reply:
Ok, that's fair. Let's not let this get out of hand but we can live with one or two of these kinds of things for awhile.